packages/devtools-reps/src/object-inspector/components/ObjectInspector.css

```diff
@@ -28,17 +28,23 @@
 }
 
 .tree.object-inspector .block .object-label:before {
-  content: "☲ ";
-  font-size: 1.1em;
+  content: "☲";
+  font-size: 1em;
+  display: inline-block;
```

Comment:
Do we need inline-block here?
After you switch to inline-block, this pseudo-element will start to impact the parent's height, so you need a bunch of vertical-align and negative margin tricks. If we can keep `display:inline`, it might work out better?

Reply:
Good call. Took it out and used `display: inline` to make it simpler and more reliable.